src/containers/shared/utils.js

```diff
@@ -187,8 +190,21 @@ export const formatLargeNumber = (d = 0, digits = 4) => {
     };
   }
 
+  while (numberCopy < 1 && variableDigits < 20) {
+    numberCopy *= 10;
+    numberOfZeros += 1;
+    variableDigits = numberOfZeros > variableDigits - 1 ? variableDigits + 1 : variableDigits;
+  }
```

Comment:
Leave a comment here that explains what this is doing

Reply:
Counting the number of zeros after the decimal point and displaying two non-zero digits (eg. 0.00000023/45/50) at the end. Also, we can display a maximum of 20 digits.

Reply:
No, leave a comment in the code